release.mk

```diff
@@ -165,7 +165,9 @@ common-changelog:
 	mv changelogs/head.asciidoc changelogs/$(VERSION).asciidoc
 	$(SED) 's#head#$(VERSION)#gI' changelogs/$(VERSION).asciidoc
 	$(SED) -E -e 's#(\...)main#\1$(VERSION)#g' changelogs/$(VERSION).asciidoc
-	awk "NR==5{print \"\n* <<release-notes-$(VERSION).0>>\n\n[float]\n[[release-notes-$(VERSION).0]]\n=== APM version $(VERSION).0\"}1" changelogs/$(VERSION).asciidoc > changelogs/$(VERSION).asciidoc.new
+	$(SED) 's#https://github.com/elastic/apm-server/compare/.*##gI' changelogs/$(VERSION).asciidoc
```

Comment:
This sed won't remove the two empty lines. But if for anything is automated then changes should also be reflected in the `awk` since it uses a fixed line. (number 5)

Reply:
This is not the case anymore... I used a different approach, see https://github.com/elastic/apm-server/pull/13065/commits/55d1fbf24b2fb495850e2a169a748cc46fb9c551

therefore no more sed/awk craziness but templates